src/main/java/org/opensrp/repository/postgres/mapper/custom/xml/CustomLocationMetadataMapper.xml

```diff
@@ -360,4 +360,38 @@
 
 	</select>
 
+	<select id="selectChildrenIds"
+			resultMap="org.opensrp.repository.postgres.mapper.LocationMapper.BaseResultMap">
```

Comment:
Return a list of String. return type should be string

Reply:
Done